teams/mods-mediators.toml

```diff
@@ -26,12 +28,12 @@ orgs = ["rust-lang", "rust-lang-nursery"]
 [rfcbot]
 label = "T-moderation"
 name = "Moderation"
-ping = "rust-lang/moderation"
+ping = "rust-lang/mods-core"
 
 [website]
 page = "moderation"
 name = "Moderation team"
```

Comment:
The website literally is still called Moderation team with zero changes. Changing the name of a rust module for a stable feature doesn't need T-lang discussion either

Reply:
Do you disagree that calling mods-venue "T-mods" raises some possibility for confusion here?

Maybe I'm too close to it.  If you think this is totally clear from governance perspective, I won't stand in your way here.